aiida/cmdline/commands/cmd_devel.py

```diff
@@ -20,26 +22,36 @@ def verdi_devel():
 
 
 @verdi_devel.command('check-load-time')
-def devel_check_load_time():
+@click.option('-v', '--verbose', is_flag=True, help='Increase verbosity')
```

Comment:
I think there is a verbosity option one can simply import... anyhow we'll fix this soon hopefully

Reply:
Sure, although this is a dev command, so I'm not really to bothered about it being that formal